mne/viz/evoked.py

```diff
@@ -1191,3 +1191,342 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+        styles[condition]['linestyle'] = styles[condition].get(
+            'linestyle', linestyles[condition])
+
+    # the actual plot
+    any_negative, any_positive = False, False
+    for condition in conditions:
+        if picks is not None:
+            if ch_type == 'grad':
+                d = ((_merge_grad_data(evokeds[condition]
+                      .data[pairpicks, :][picks, :])).T * scaling).mean(-1)
+            else:
+                d = ((evokeds[condition].data[picks, :]).T * scaling).mean(-1)
+        else:
+            d = ((evokeds[condition].data).T * scaling).std(-1)
+        if ci and picks is not None:
+            sem_ = sem_array[condition]
+            ax.fill_between(times, sem_[0].flatten() * scaling,
+                            sem_[1].flatten() * scaling,
+                            color=styles[condition]['c'], alpha=.333)
+
+        ax.plot(times, d, zorder=10, label=condition, **styles[condition])
+        if any(d > 0):
+            any_positive = True
+        if any(d < 0):
+            any_negative = True
+
+    # truncate the y axis ... yes, it's that complicated
+    orig_ymin, orig_ymax = ax.get_ylim()[0], ax.get_ylim()[-1]
+    if not any_positive:
+        orig_ymax = 0
+    if not any_negative:
+        orig_ymin = 0
+
+    ax.set_ylim(orig_ymin if ymin is None else ymin,
+                orig_ymax if ymax is None else ymax)
+
+    fraction = 2 if ax.get_ylim()[0] >= 0 else 3
+
+    if truncate_yaxis:
+        abs_lims = (orig_ymax if orig_ymax > np.abs(orig_ymin)
+                    else np.abs(orig_ymin))
+        ymin_, ymax_ = (-(abs_lims // fraction), abs_lims // fraction)
+        # custom ymin/ymax must overwrite everything
+        if ymin is not None and ymin > ymin_:
+            ymin_ = ymin
+        if ymax is not None and ymax < ymax_:
+            ymax_ = ymax
+        ax.set_yticks((ymin_ if any_negative else 0,
+                       ymax_ if any_positive else 0))
+        ymin_bound, ymax_bound = (-(abs_lims // fraction),
+                                  abs_lims // fraction)
+        if ymin is not None and ymin > ymin_bound:
+            ymin_bound = ymin
+        if ymax is not None and ymax < ymax_bound:
+            ymax_bound = ymax
+        ax.spines['left'].set_bounds(ymin_bound, ymax_bound)
+    else:
+        ymax_bound = ax.get_ylim()[-1]
+    y_range = -np.subtract(*ax.get_ylim())
+
+    # style the spines/axes
+    ax.set_title(", ".join(ch_names) if title is None else title)
+    ax.spines["top"].set_position('zero')
+    ax.spines["top"].set_smart_bounds(True)
+
+    ax.tick_params(direction='out')
+    ax.tick_params(right="off")
+
+    current_ymin = ax.get_ylim()[0]
+
+    # plot v lines
+    if invert_y is True and current_ymin < 0:
+        upper_v, lower_v = -ymax_bound, ax.get_ylim()[-1]
+    else:
+        upper_v, lower_v = ax.get_ylim()[0], ymax_bound
+    for x in vlines:
+        ax.vlines(x, upper_v, lower_v,
```

Comment:
IIRC I tried that and it didn't work with the ancient Python's MPL or something like that.